src/common.c

```diff
@@ -1239,7 +1239,11 @@ psf_memset (void *s, int c, sf_count_t len)
 typedef SF_CUES_VAR (1) SF_CUES_1 ;
 typedef SF_CUES_VAR (2) SF_CUES_2 ;
 
-#define SF_CUES_VAR_SIZE(count)	(sizeof (SF_CUES_1) + count * (sizeof (SF_CUES_2) - sizeof (SF_CUES_1)))
+/* calculate size of SF_CUES struct given number of cues */
+#define SF_CUES_VAR_SIZE(count)	(sizeof (SF_CUES_1) + ((int) (count) - 1) * (sizeof (SF_CUES_2) - sizeof (SF_CUES_1)))
```

Comment:
I've commit a better way to calculate that to master. You should rebase against that.